### PR TITLE
Detecting of the "MediaStream has no audio track"

### PR DESCRIPTION
**Pull request by anhr**
I see the

soundmeter.js:50 DOMException: Failed to execute
'createMediaStreamSource' on 'AudioContext': MediaStream has no audio
track(…)

error if I have video input device and NOT have audio input device in my
system. I have added the try catch block into
SoundMeter.prototype.connectToSource function for resolving of the
problem.

### CODE REVIEW

**Comment by anhr** [comment]
I signed it!

File: src/content/getusermedia/volume/js/main.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ try {
 // Put variables in global scope to make them available to the browser console.
 var constraints = window.constraints = {
   audio: true,
-  video: false
+  video: true
```
Comment (KaptenJansson) [inline src/content/getusermedia/volume/js/main.js]: This should be false still.

**Comment by KaptenJansson** [comment]
Thanks for the contribution, will take a look.

File: src/content/getusermedia/volume/js/main.js [hunk 2]
```diff
@@ -36,16 +36,22 @@ function successCallback(stream) {
...
-
-  setInterval(function() {
-    instantMeter.value = instantValueDisplay.innerText =
-        soundMeter.instant.toFixed(2);
-    slowMeter.value = slowValueDisplay.innerText =
-        soundMeter.slow.toFixed(2);
-    clipMeter.value = clipValueDisplay.innerText =
-        soundMeter.clip;
-  }, 200);
+  soundMeter.connectToSource(stream, function (e) {
```
Comment (KaptenJansson) [inline src/content/getusermedia/volume/js/main.js]: Please fix all the style issues as you can see in the test run: https://travis-ci.org/webrtc/samples/jobs/117060901.

**Comment by KaptenJansson** [comment]
I just realized this demo will should never continue if no audio devices are present due to getUserMedia will fire the error callback. I assume you are using the soundmeter.js somewhere else and want to add error handling to make it more robust?

**Comment by anhr** [comment]
Yes of course I use soundmeter.js in my project. I have audio: true and video: true and I want to see the sound value.  The soundmeter.js works exellent if I have microphone and web cam in my system. But I see an exception if microphone is not exists. I want to inform user about exception and ask to install a microphone. I think it is trivial case and many developers wants to catch this exception.

**Comment by KaptenJansson** [comment]
Ping, could you address my comment and then merge with gh-pages branch, thanks!

**Comment by KaptenJansson** [comment]
Ping

**Comment by anhr** [comment]
I has fixed all the style issues.
See https://travis-ci.org/webrtc/samples/jobs/124992920

**Comment by KaptenJansson** [comment]
You also need to merge with gh-pages branch please.

**Comment by anhr** [comment]
I think I has  successfully merged with gh-pages branch. Correct?

File: src/content/getusermedia/volume/js/main.js (hunk 1 above)
Comment (KaptenJansson) [inline src/content/getusermedia/volume/js/main.js]: You still haven't changed this. This should not require video.

Comment (anhr) [inline src/content/getusermedia/volume/js/main.js]: ready

**Comment by KaptenJansson** [comment]
LGTM. I will deal with the FF beta failure.

**Pull request merged**